README.md

```diff
@@ -14,53 +14,56 @@ The third sub-system is the `web-server` JS and CSS files, used to display
 various graphs and results, and any other content generated by the
 `pbench-agent` during benchmark and tool post-processing steps.
 
-The pbench Dashboard code lives in its own [repository](https://github.com/distributed-system-analysis/pbench-dashboard).
+The pbench Dashboard code lives in its own [repository](
+https://github.com/distributed-system-analysis/pbench-dashboard).
 
 ## How is it installed?
 Instructions on installing `pbench-agent`, can be found
-in the [Pbench agent installation](http://distributed-system-analysis.github.io/pbench/doc/agent/installation.html) guide.
+in the Pbench Agent [Getting Started Guide](
+https://distributed-system-analysis.github.io/pbench/start.html).
 
 For Fedora, CentOS, and RHEL users, we have made available [COPR
 builds](https://copr.fedoraproject.org/coprs/ndokos/pbench/) for the
-`pbench-agent`, `configtools`, `pbench-server`, `pbench-web-server`
-and some benchmark and tool packages.
+`pbench-agent`, `pbench-server`, `pbench-web-server`, and some benchmark and
+tool packages.
 
-This assumes that somebody has already installed the server bits. The
-procedure to do that is described in the [Pbench Server
-Installation](http://distributed-system-analysis.github.io/pbench/doc/server/installation.html)
-guide.
+Install the `pbench-web-server` package on the machine from where you want to
+run the `pbench-agent` workloads, allowing you to view the graphs before
+sending the results to a server, or even if there is no server configured to
+send results.
 
-You can install the `web-server` sub-system on the machine from where you
-want to run the `pbench-agent`. That allows you to view the graphs before
-sending the results to the server, or even if there is no server to which to
-send the results. See the [Pbench web server installation](http://distributed-system-analysis.github.io/pbench/doc/server/pbench-web-server.html) guide.
-
-You might want to browse through the [rest of the documentation](http://distributed-system-analysis.github.io/pbench/doc/).
+You might want to consider browsing through the [rest of the documentation](
+https://distributed-system-analysis.github.io/pbench/doc.html).
 
 ## How do I use pbench?
-Refer to the [Pbench User Guide](http://distributed-system-analysis.github.io/pbench/doc/agent/user-guide.html).
+Refer to the [Pbench Agent Getting Started Guide](
+https://distributed-system-analysis.github.io/pbench/start.html).
 
-TL;DR? See [TL;DR - How to set up pbench and run a benchmark
-](http://distributed-system-analysis.github.io/pbench/doc/agent/user-guide.html#org9c5bc26) of the
-[Pbench User
-Guide](http://distributed-system-analysis.github.io/pbench/doc/agent/user-guide.html) for a
+TL;DR? See [TL;DR - How to set up the `pbench-agent` and run a benchmark
+](https://distributed-system-analysis.github.io/pbench/doc.html#how) of the
+[Pbench Agent Getting Started Guide](
+https://distributed-system-analysis.github.io/pbench/doc.html) for a
```

Comment:
The TL;DR in the previous link follows this one after a brief intro paragraph. That doesn't make it wrong; but anyone clicking through the links in sequence (like I just did) is going to think it looks a bit weird for the second link to just jump a few lines back from the first.

Reply:
Once we have addressable tabs in the "Getting Started" page (see #1907), we can change the links to go to the separate sections, allowing us to re-arrange the content a bit.  But that is another effort.

This is actually wrong, in that the "Getting Started Guide" link is not the main documentation, so I'll just drop it.